src/components/nc-phone-input.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div
+    :class="['nc-phone-input', wrapperClasses, { disabled: disabled }]"
+    :selected-country="selectedCountry">
+    <div
+      class="nc-phone-input__country-code"
+      @click="showCountryCodesList">
+      <!-- emoji -->
+      <img
+        v-if="enabledFlags"
+        class="nc-phone-input__flag"
+        :src="'data:image/png;base64, ' + selectedCountry.flag" />
+      <!-- country code -->
+      <span v-if="enabledCountryCode" class="nc-phone-input__code-number">{{ selectedCountry.dialCode }}</span>
+    </div>
+    <!-- TODO: Replace for nc-input -->
+    <input
+      ref="input"
+      :class="['nc-phone-input__phone', inputClasses]"
+      v-model="phone"
+      type= "tel"
+      :placeholder="placeholder"
+      :disabled="disabled"
+      :required="required"
+      :name="name"
+      :maxlength="maxLength"
+      @blur="onBlur"
+      @input="onInput" />
+  </div>
+</template>
+
+<script>
+/**
+ * Code based on https://github.com/EducationLink/vue-tel-input
+ * Credit to https://github.com/iamstevendao
+ */
+import { formatNumber, AsYouType, isValidNumber } from 'libphonenumber-js'
+
+export default {
+  name: 'nc-phone-input',
+  props: {
+    value: {
+      type: String
+    },
+    placeholder: {
+      type: String,
+      default: 'Enter a phone number'
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    },
+    disabledFormatting: {
+      type: Boolean,
+      default: false,
+    },
+    invalidMsg: {
+      default: 'Wrong phone number',
+      type: String,
+    },
+    enabledCountryCode: {
```

Comment:
```suggestion
    countryCodeEnabled: {
```

De la otra manera parece que indica el country code que se ha habilitado, yo normalmente si no tengo claro como llamarlas lo planteo como pregunta `is my prop <status>?`

Reply:
Done